### DISCUSSION TITLE
Privacy Policy of SuperSlicer

### DISCUSSION THREAD

**Discussion by malteroggenberg** [supermerill/SuperSlicer#4068] [Q&A] [answered]
Hi,
seems like a strange question for an open source Project, but I'm working on a school project about Slicers also regarding SuperSlicer.

I was wondering if SuperSlicer collects or stores any personal data during the use of the slicer or if SuperSlicer collects any type of System related data like usage or Hardware. Ty guys in advance!

**Comment by supermerill** [comment, marked as answer]
None I know.
If prusa made something, I should have removed it.
Something may have slight through, but unlikely.
